AuthenticationDemo/ExampleSessionService.swift

```diff
@@ -67,14 +69,45 @@ public final class ExampleSessionService: SessionServiceType {
         })
     }
     
-    private func failureSignalProducer(dispatchTime: dispatch_time_t) -> SignalProducer<ExampleUser, SessionServiceError> {
+    private func logInFailureSignalProducer(dispatchTime: dispatch_time_t) -> SignalProducer<ExampleUser, SessionServiceError> {
```

Comment:
I would not name this with SignalProducer, just logInFailure(...) -> ....
You could leave it like this though if you prefer it this way